test/runtests.jl

```diff
@@ -339,3 +339,14 @@ end
     @test_nowarn @inferred transform(t, x)
     @test_nowarn @inferred transform_and_logjac(t, x)
 end
+
+@testset "positive definite cholesky factor" begin
+    t = PosDefCholeskyFactor(4)
+    d = dimension(t)
+
+    v = randn(d)
+    U = transform(t, v)
+    @test U <: UpperTriangular
+    @test size(U) = (4,4)
+    @test inverse(t,U) ≈ v
+end
```

Comment:
Please 

- also test for `U` being positive (semi) definite (eg just check the diagonal),
- check the log jacobian using ForwardDiff (you find examples of this in the tests),
- test that the results are `@inferred`

Happy to help if necessary.